esp/platform/esp.hpp

```diff
@@ -93,6 +93,7 @@ typedef enum LogRequest_
 
 #define TXSUMMARY_OUT_TEXT      0x00000001
 #define TXSUMMARY_OUT_JSON      0x00000002
```

Comment:
Why'd we choose that higher (out of order) bit to signify FWD_OTEL? perhaps a comment would be helpful if we're reserving the lower bits for specific TXSUMMARY_OUT types, and want to use the last few bits for another set of flags

Reply:
We support serialization of summary content using the two output styles. I'm piggybacking on that mechanism to cause cumulative timer forwarding at the end of a transaction and thought the value separation for the new bit would be an additional clue that it isn't a serialization output. Plus, as the span takes over I can envision adding an otel output style that does request serialization in the future.

Reply:
Perfect. Let's not just leave clues, let's be explicit to avoid any potential future misunderstandings.